### PR TITLE
Feature/search list directories

### PR DESCRIPTION
**Pull request by mauricioZelaya**

### CODE REVIEW

File: src/com/jalasoft/search_files/search/search_engine.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# def search_file(path, file_name):
+#     for root, dirs, files in os.walk(path):
+#         print(root)
+#         # if file_name in files:
+#         #     print("file path directory: %s" % root)
+#         #     print(dirs)
+#         #     print(files)
+#         #     print("exist!!!")
+
+
```
Comment (Sl3v7n) [inline src/com/jalasoft/search_files/search/search_engine.py:19]: This should not be included in the code at least is part of the documentation

File: src/com/jalasoft/search_files/search/search_engine.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# search_file("D:", "Behave_1.pptx")
+path = "D:\MauricioZ\\"
```
Comment (Sl3v7n) [inline src/com/jalasoft/search_files/search/search_engine.py:21]: Is it strictly necessary to use your own path??

**Review by Sl3v7n — CHANGES_REQUESTED** [review]
it need changes

File: src/com/jalasoft/search_files/search/search_engine.py (hunk 1 above)
Comment (mauricioZelaya) [reply src/com/jalasoft/search_files/search/search_engine.py:19]: everything will be refactor on the next branch this was uploaded as an example of use

**Review by Sl3v7n — APPROVED** [review]

**Pull request merged**